src/shared/faas-debugger.ts

```diff
@@ -331,11 +331,11 @@ ${originalCode}
     /* istanbul ignore else */
     if (updatedCode.includes('This is an auto generated code')) {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      const [_1, originalCode1] = updatedCode.split(`// Rewire require
+      const [, originalCode1] = updatedCode.split(`// Rewire require
```

Comment:
Writing it like this makes it perfectly clear that you are not interested in the first parameter. Especially given that you are ignoring it.

```suggestion
      const [_, originalCode1] = updatedCode.split(`// Rewire require
```